git_machete/client.py

```diff
@@ -1029,7 +1048,13 @@ def print_line_prefix(branch_: LocalBranchShortName, suffix: str) -> None:
             else:
                 current = bold(branch)
 
-            anno: str = f"  {dim(self.__annotations[branch])}" if branch in self.__annotations else ""
+            anno: str = ''
+            if branch in self.__annotations:
```

Comment:
I'd consider moving such rendering logic to `Annotation` class, to offload the Client itself